app/components/projects/case.tsx

```diff
@@ -1,108 +1,107 @@
+import { ArrowLeftIcon } from "@radix-ui/react-icons";
+import { OpenLink } from "../icons";
+
 export interface CaseProps {
-  logo?: string;
-  title?: string;
-  description?: string;
-  problem_statement?: string;
-  solution?: string;
-  website_image?: string;
-  link?: string;
+  logo?: string | null;
```

Comment:
why `| null`? If prismic returns null values by default ( i forgot if they do), then it would make sense for it to be either only `string | null` or kept as `string | undefined` and set to undefined wherever it is passed if it is null.

Side note: `link?: string` directly translates to `link: string | undefined`. [docs](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties)